src/pages/workshop/nllp-2022.md

```diff
@@ -32,6 +32,428 @@ The workshop proceedings are available [here](https://aclanthology.org/events/em
 
 The recording of the workshop is available [here](https://www.youtube.com/watch?v=cdHE7u9vfSk&ab_channel=NaturalLegalLanguageProcessingWorkshop).
 
-The program is available [here](https://nllpw.org/workshop/program/).
+[Michael Livermore] (https://www.law.virginia.edu/faculty/profile/mal5un/2457619) (University of Virginia School of Law) was the invited speaker.
 
-Michael Livermore (University of Virginia School of Law) was the [invited speaker](https://nllpw.org/workshop/speakers/).
+
+---
```

Comment:
Check the preview: https://deploy-preview-60--adorable-spider-f59d7.netlify.app/workshop/nllp-2022

This metadata section needs to be removed from the body and only kept at the top